waffle/templatetags/waffle_tags.py

```diff
@@ -81,3 +87,32 @@ def render(self, context):
 @register.tag
 def wafflejs(parser, token):
     return InlineWaffleJSNode()
+
+
+@register.filter(name="switch_is_active")  # type: ignore[misc]
+def switch_is_active(switch_name: str) -> bool:
+    """
+    This template filter works like the switch tag, but you can use it within
+    if statement conditions in Django templates. Example: `{% if "switch_name"|switch_is_active %}`.
+    """
+    return waffle_switch_is_active(switch_name)
+
+
+@register.simple_tag  # type: ignore[misc]
```

Comment:
question: should this be a filter like the switch and sample above and below ? context: I don't use django templates so apologies if I'm missing something obvious

Reply:
Good question @dancergraham , so since there are more than 2 arguments with differing types for this function, we cannot use a template filter for it, therefore, we use a simple tag and set the variable in the django template.

```
{% flag_is_active "flag" request True as is_flag_active %}
{% if is_flag_active %}
  flag_is_active on
{% else %}
  flag_is_active off
{% endif %}
```

Reply:
Sadly it is not as clean of an implementation, and there are forums about this exact issue dating back to 2007 (https://code.djangoproject.com/ticket/1199)